app/es6/jarb-formly-field-transformer.factory.js

```diff
@@ -86,10 +86,14 @@ angular.module('jarb-angular-formly')
      * @return {[field]}        A transformed array of formly fields
      */
     function transform(fields, model, options) {
+      if (entityNameIsDefined(options) === false) {
+        return fields;
+      }
+
       const constraints = constraintsStore.getConstraints();
 
       return fields.map((field) => {
-        if (entityNameIsDefined(options) === false) {
+        if (_.get(field, 'data.ignoreDatabaseConstraints') === true) {
```

Comment:
Rename this to 'ignoreJarbConstraints', this makes this a little bit more clear.